docs/self-managed/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -5,110 +5,279 @@ description: "A guide to creating backup of a running Zeebe cluster."
 keywords: ["backup", "backups"]
 ---
 
-:::note
-This API is subject to change.
-:::
+A backup of a Zeebe cluster consists of a consistent snapshot of all partitions.
+The backup is taken asynchronously in the background while Zeebe is processing.
+Thus, the backups can be taken with minimal impact on normal processing.
+The backups are stored externally and can be used to restore a cluster in case of failures that leads to full data loss or data corruption.
+
+Zeebe provides a Rest API to create backups, and query and manage existing backups.
+The API is accessible via the management port of the gateway.
+
+## Prerequisites
 
-A backup of a Zeebe cluster consists of a consistent snapshot of all partitions. The backup is stored externally. The backup is taken asynchronously in the background while Zeebe is processing. Thus, the backups can be taken with minimal impact on normal processing.
+To use the backup feature in Zeebe, the following configurations must be provided.
 
-Backup of a running Zeebe cluster can be taken using the following rest APIs. The API is accessible via the management port of the gateway.
+- Enable backups by setting the flag `ZEEBE_BROKER_EXPERIMENTAL_FEATURES_ENABLEBACKUP` to `true`.
+- The backup management API is available via management port of the gateway. Ensure the configuration `MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE` include "backups". Set `MANAGEMENT_ENDPOINTS_BACKUPS_ENABLED` to `true`.
+- Backup is stored in an external storage. This must be configured before starting the Zeebe cluster.
```

Comment:
:upside_down_face: I would split this away from the prerequisites, or at least make it a sub-step (with its own, lower-level, header), specifically around configuring the external backup store. Soft suggestion, but I think it's a slightly more "complex" point than the previous ones, and it sets up a separation for further storage plugins in the future.